### PR TITLE
use xccdf variable in audit_audispd_network_failure_action

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- use XCCDF variable in the rule description and ocil

#### Rationale:

- STIG effort

Reference: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_7_security_technical_implementation_guide/V-73163?version=v2r7

### CODE REVIEW

File: linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_network_failure_action/rule.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ description: |-
     <tt>halt</tt>. For certain systems, the need for availability
     outweighs the need to log all actions, and a different setting should be
     determined.
+    This profile configures the <i>action</i> to be {{{ sub_var_value("var_audispd_network_failure_action") }}}.
```
Comment (jan-cerny) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_network_failure_action/rule.yml:17]:
```suggestion
    This profile configures the <i>ACTION</i> to be <tt>{{{ sub_var_value("var_audispd_network_failure_action") }}}</tt>.
```

Comment (matejak) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_network_failure_action/rule.yml:17]: and you can `s/sub_var_value/xccdf_value/` (`sub_var_value` still works though).

**Comment by vojtapolasek** [comment]
Thank you, I changed the macro used.

**Pull request merged**